crypto/evp/evp_fetch.c

```diff
@@ -306,7 +306,9 @@ void *evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
 #else
         ERR_raise_data(ERR_LIB_EVP, code,
                        "%s, Algorithm (%s), Properties (%s)",
-                       (openssl_ctx_is_default(libctx)
+                       (openssl_ctx_is_global_default(libctx)
+                        ? "Global default library context"
+                        : openssl_ctx_is_default(libctx)
                         ? "Default library context"
```

Comment:
Shouldn't his be indented?

Reply:
"Should" is a matter of taste in this case.  We have said nothing about this particular type of condition cascade.  That being said, I can add parentheses around the last condition if you think that's better.

Reply:
Ping?